code/modules/antagonists/conspirator/conspirator.dm

```diff
@@ -14,20 +14,22 @@
 		if (!src.meeting_point)
 			src.meeting_point = "Your initial meet-up point is <b>[pick("the chapel", "the bar", "disposals", "the arcade", "the escape wing", "crew quarters", "the pool", "the aviary")].</b>"
 
-		if (!src.conspirator_list)
-			src.conspirator_list = "The conspiracy consists of: "
+		if (!src.conspirators)
+			src.conspirators = list()
 
 		src.owner = new_owner
-		if (src.owner.assigned_role == "Clown")
-			src.conspirator_list += "<b>a Clown</b>, "
-		else
-			src.conspirator_list += "<b>[src.owner.current.real_name]</b>, "
+		src.conspirators += src.owner
 
 		. = ..()
 
 	give_equipment()
-		src.owner.store_memory(src.meeting_point)
-		src.owner.store_memory(src.conspirator_list)
+		SPAWN(1 SECOND)
+			var/conspirator_list = src.generate_conspirator_list()
```

Comment:
Won't this result in newly joining conspirators having different lists to older ones? Better to add code to `show_memory` to generate it each time (or better yet, make `show_memory` call a proc on the antag datum to ask for any memory contents it might need to add)

Reply:
It will, but not for roundstart conspirators, as `SPAWN(1 SECOND)` permits all minds to receive a conspirator antagonist datum prior to generating conspirator lists. This was done to avoid scope creep, as ideally a dynamic memory system should be atomised into another PR; I would rather implement a reusable system as opposed to adding snowflake checks to cater to a specific use case.

Reply:
alright, I'll never argue against atomizing more!